src/SoundList.js

```diff
@@ -9,7 +9,18 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+const SoundList = ({
+  tracks,
+  onSoundClick,
+  header,
+  currentTrackId,
+  className = "",
+}) => {
+  const selectedTrackClassName = "bg-secondary text-primary";
+  const unSelectedTrackClassName = "hover:bg-secondary hover:text-primary";
```

Comment:
I think this can be inlined instead of conditionally adding the class. Event if we apply this to a selected class, it shouldn't make a difference. Inline has the benefit of simplifying the code

Reply:
Definitely! I added a couple of comments clarifying what the more succinct code does; hopefully that'll mitigate any confusion.